docs/source/routing/self-hosted/index.mdx

```diff
@@ -37,6 +37,7 @@ Follow our [Kubernetes quickstart](/graphos/routing/kubernetes/quickstart) to de
 Apollo provides the Apollo Runtime Container, which bundles all that's required to run the Apollo Runtime in one place. This includes the Apollo Router and the [Apollo MCP Server](/apollo-mcp-server).
 
 The images are available via GitHub, downloadable from `ghcr.io/apollographql/apollo-runtime` and the [Apollo Runtime Container repository](https://github.com/apollographql/apollo-runtime).
+Alternatively you can download the images from DockerHub at `apollograph/apollo-runtime`.
```

Comment:
Is this meant to exist soon? https://hub.docker.com/r/apollograph/apollo-runtime?   (Can we link to that?, and also fix the other link.  I think it's a bit confusing to not link to the pages which have the full context of the versions, etc, particularly since we don't really want people using `latest`, I'd guess)

Reply:
Yer that makes sense, I'm just getting that all spun up so will do when it's available then the docs can go out after the fact 👍